.github/workflows/main-workflow.yml

```diff
@@ -13,8 +13,15 @@ jobs:
         with: 
           node-version: '14'
 
-      - name: Install Dependncies
-        run:  sudo npm install
+      - name: Install Dependencies
+        run:  npm install mocha

```

Comment:
If you include `mocha` in `devDependencies` of package.json, this line will not be required. Try it out.